.ci/run.sh

```diff
@@ -26,6 +26,9 @@ case "${CI_JOB}" in
 		echo "INFO: Running vcpus test"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make vcpus"
 		echo "INFO: Skipping pmem test: Issue: https://github.com/kata-containers/tests/issues/3223"
+		echo "INFO: Running sandbox cgroup with stability test"
```

Comment:
could this be rephrased to "Running stability test with sandbox_cgroup_only"?

Reply:
@chavafg changes applied